test/REGRESSIONS

```diff
@@ -497,7 +485,7 @@ gbt is working on getting rid of this
 
 signal 11 (first seen 2014-03-02)
 ---------------------------------
-[Error matching compiler output for release/examples/benchmarks/shootout/meteor-fast]
+[Error matching program output for release/examples/benchmarks/shootout/meteor-fast]
```

Comment:
Is it accurate that the signal 11 in the program output was first seen 2014-03-02 (i.e., someone mis-classified it as compiler output), or is that more recent?  If more recent, it seems like we might want to split this out (or put the dates by the test failures themselves).